docs/content/stable/yugabyte-platform/back-up-restore-universes/configure-backup-storage.md

```diff
@@ -80,23 +80,55 @@ You can configure Google Cloud Storage (GCS) as your backup target, as follows:
 
 1. Navigate to **Configs > Backup > Google Cloud Storage**.
 
-2. Click **Create GCS Backup** to access the configuration form shown in the following illustration:
+1. Click **Create GCS Backup** to access the configuration form shown in the following illustration:
 
-    ![GCS Configuration](/images/yp/cloud-provider-configuration-backup-gcs.png)
+    ![GCS Configuration](/images/yp/cloud-provider-configuration-backup-gcs-stable.png)
 
-3. Use the **Configuration Name** field to provide a meaningful name for your backup configuration.
+1. Use the **Configuration Name** field to provide a meaningful name for your backup configuration.
+
+1. Complete the **GCS Bucket** field.
```

Comment:
@Arpit-yb - what goes in this field ?

Reply:
We provide the cloud location here. For GCS it would be the GCS bucket + some optional subdirectory inside that bucket. For eg: `gs://gcp-bucket/test_backups`